datasets/pubmed_qa/pubmed_qa.py

```diff
@@ -0,0 +1,244 @@
+The task of PubMedQA is to answer research questions with yes/no/maybe (e.g.: Do preoperative
+statins reduce atrial fibrillation after coronary artery bypass grafting?) using the corresponding abstracts.
+PubMedQA has 1k expert-annotated, 61.2k unlabeled and 211.3k artificially generated QA instances.
+Each PubMedQA instance is composed of (1) a question which is either an existing research article
+title or derived from one, (2) a context which is the corresponding abstract without its conclusion,
+(3) a long answer, which is the conclusion of the abstract and, presumably, answers the research question,
+and (4) a yes/no/maybe answer which summarizes the conclusion.
+PubMedQA is the first QA dataset where reasoning over biomedical research texts, especially their
+quantitative contents, is required to answer the questions.
+"""
+
+
+_HOMEPAGE = "https://pubmedqa.github.io/"
+
+_LICENSE = """\
+MIT License
+Copyright (c) 2019 pubmedqa
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {
+    "ori_pqal": "https://raw.githubusercontent.com/pubmedqa/pubmedqa/master/data/ori_pqal.json",
+    "ori_pqau": "https://drive.google.com/uc?export=download&id=1RsGLINVce-0GsDkCLDuLZmoLuzfmoCuQ",
+    "ori_pqaa": "https://drive.google.com/uc?export=download&id=15v1x6aQDlZymaHGP7cZJZZYFfeJt2NdS",
+}
+
+
+class PubMedQAConfig(datasets.BuilderConfig):
+    """BuilderConfig for PubMedQA"""
+
+    def __init__(self, **kwargs):
+        """
+        Args:
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super(PubMedQAConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+
+
+class PubMedQA(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class PubmedQA(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `pubmed_qa`